src/python/PSetTweaks/WMTweak.py

```diff
@@ -119,7 +125,7 @@ def lfnGroup(job):
     default both to 0. The result will be a 5-digit string.
     """
     modifier = str(job.get("agentNumber", 0))
-    jobLfnGroup = modifier + str(job.get("counter", 0) / 1000).zfill(4)
+    jobLfnGroup = modifier + str(old_div(job.get("counter", 0), 1000)).zfill(4)
```

Comment:
I'd actually use the division from futurize (this place is expecting an integer division `//`)

Reply:
Do you mean that we could just use the `//` operator, as suggested by [futurize cheatsheet](https://python-future.org/compatible_idioms.html#division)